scripts/docker_release/algod_docker_init.sh

```diff
@@ -0,0 +1,39 @@
+#!/usr/bin/env bash
+
+GENESIS_FILE="/root/node/data/genesis.json"
+CONFIG_FILE="/root/node/data/config.json"
+
+echo "initializing docker container for algod network: '${ALGORAND_NETWORK}'"
+
+if [[ ${ALGORAND_NETWORK} != "devnet" && ${ALGORAND_NETWORK} != "testnet" && ${ALGORAND_NETWORK} != "mainnet"  ]]
+then
+    echo "error, unrecognized network: ${ALGORAND_NETWORK}, please specify one of: devnet, testnet, mainnet"
+    echo "defaulting to: 'devnet'"
```

Comment:
We should default to testnet - this is for public consumption so we don't want the default to be devnet.

Reply:
ok, set default to testnet.